src/main.ts

```diff
@@ -74,3 +72,34 @@ function isGhes(): boolean {
   );
   return ghUrl.hostname.toUpperCase() !== 'GITHUB.COM';
 }
+
+function resolveVersionInput(): string {
+  let version = core.getInput('node-version') || core.getInput('version');
+  const versionFileInput = core.getInput('node-version-file');
+
+  if (version && versionFileInput) {
+    core.warning(
+      'Both node-version and node-version-file inputs are specified, only node-version will be used'
+    );
+  }
+
+  if (version) {
+    return version;
+  }
+
+  if (versionFileInput) {
+    const versionFilePath = path.join(
+      process.env.GITHUB_WORKSPACE!,
+      versionFileInput
+    );
+    if (!fs.existsSync(versionFilePath)) {
+      throw new Error('The specified node version file does not exist');
```

Comment:
```suggestion
      throw new Error(`The specified node version file in the path ${versionFilePath} does not exist`);
```

Some improvement to output the full path